### PR TITLE
Added support for CIT option

### PR DESCRIPTION
**Pull request by shailesh-kalamkar**
@arpit1094 _A Please review.

### CODE REVIEW

**Comment by arpit1094** [comment]
@shailesh-kalamkar , I was going through geocoder gem code. Will it make more sense to create a setting `use_cit` like https://github.com/alexreisner/geocoder/blob/master/lib/geocoder/configuration.rb ?This will also save us the hassle of passing `cit` option everywhere.

**Comment by shailesh-kalamkar** [comment]
@arpit1094 We will need to dynamically switch the `cit` option based on the environment, so don't know how configuration will fit for it.

**Comment by arpit1094** [comment]
@shailesh-kalamkar , that can be controlled by environment files like `app_id` and `app_code`

**Comment by arpit1094** [comment]
@shailesh-kalamkar _A, please review.

File: lib/geocoder/lookups/default_lookup_methods.rb [hunk 1]
```diff
@@ -0,0 +1,43 @@
+module Geocoder::Lookup
+  module DefaultLookupMethods
```
Comment (shailesh-kalamkar) [inline lib/geocoder/lookups/default_lookup_methods.rb]: @arpit1094 This file is `here` maps related. The module name/namespace should suggest that.

**Review by shailesh-kalamkar — COMMENTED** [review]
@arpit1094 Added one comment. Please check.

**Comment by arpit1094** [comment]
@shailesh-kalamkar , done with the changes. Please review.

**Comment by shailesh-kalamkar** [comment]
@arpit1094  Looks good to me.

**Pull request merged**